src/App.jsx

```diff
@@ -0,0 +1,24 @@
+import { Routes, Route } from 'react-router-dom';
+import Book from './components/Book';
+import Nav from './components/Nav';
+import Categories from './components/Categories';
+import Addbook from './components/Addbook';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Nav />
+      <Routes>
+        <Route path="/" element={
+        [<Book name='ashe'/>,
+        <Book name='end'/>]
+        } />
```

Comment:
- [ ] I can see what you are doing here, rendering the two books right. 😂. It would be nice for you to create a BookList component, on which you receive books prop(These books will later be fetched from an API) that will then be passed to a single Book component that you import, and then, this BookList component is now used here as a route, just as you did for the categories route. 👍 
- [ ] And finally, inside of this BookList component you created, you call the Addbook component above the rendered books. 

**See the screenshot of the instructions below**
![image](https://user-images.githubusercontent.com/50941074/190445669-a7f1ca57-d4e1-41c2-9d23-b0c5e158985c.png)